docs/deprecations.md

```diff
@@ -0,0 +1,207 @@
+- Both of these now require the schema ID to be supplied.
+
+### [`store.operation.OperationChainValidator`](https://github.com/gchq/Gaffer/blob/gaffer2-1.21.1/core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/OperationChainValidator.java)
+- The method `validateViews(final Operation op, final ValidationResult validationResult, final Schema schemaNotUsed, final Store store)` has been removed. Use `validateViews(final Operation op, final User user, final Store store, final ValidationResult validationResult)` instead, passing `user` as `null` will result in the same behaviour.
+- The method `validateComparables(final Operation op, final ValidationResult validationResult, final Schema schemaNotUsed, final Store store)` has been removed. Use `validateComparables(final Operation op, final User user, final Store store, final ValidationResult validationResult)` instead, passing `user` as `null` will result in the same behaviour.
+
+### [`store.operation.handler.named.cache.NamedViewCache`](https://github.com/gchq/Gaffer/blob/gaffer2-1.21.1/core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/named/cache/NamedViewCache.java)
+- The method `deleteNamedView(final String name)` has been removed. Use `deleteNamedView(final String name, final User user)` instead, passing `user` as `null` will result in the same behaviour.
+- The method `getNamedView(final String name)` has been removed. Use `getNamedView(final String name, final User user)` instead.
+- The method `getAllNamedViews()` has been removed. Use `getAllNamedViews(final User user)` instead.
+
+### [`types.IntegerFreqMap`](https://github.com/gchq/Gaffer/blob/gaffer2-1.21.1/core/type/src/main/java/uk/gov/gchq/gaffer/types/IntegerFreqMap.java)
+- This class has been removed.
+- Use `uk.gov.gchq.gaffer.types.FreqMap` instead, this is identical except for using Long rather than Integer.
+
+#### [`types.function.IntegerFreqMapAggregator`](https://github.com/gchq/Gaffer/blob/gaffer2-1.21.1/core/type/src/main/java/uk/gov/gchq/gaffer/types/function/IntegerFreqMapAggregator.java)
+- This class has been removed.
+- Use `uk.gov.gchq.gaffer.types.function.FreqMapAggregator` instead.
+
+#### [`serialisation.IntegerFreqMapSerialiser`](https://github.com/gchq/Gaffer/blob/gaffer2-1.21.1/core/type/src/main/java/uk/gov/gchq/gaffer/serialisation/IntegerFreqMapSerialiser.java)
+- This class has been removed.
+- Use `uk.gov.gchq.gaffer.serialisation.FreqMapSerialiser` instead.
+
+### Serialisers
+
+#### Preservation of ordering
+When using an ordered store (such as Accumulo), all serialisers used on vertices must preserve order. As such, `compactRaw` serialisers (which do not preserve order) cannot be used on vertices in ordered stores. However, when preserving order is not required `compactRaw` serialisers are the most effective solution. [See the schemas documentation for more detail](https://gchq.github.io/gaffer-doc/v1docs/getting-started/developer-guide/schemas.html#serialisers).
```

Comment:
Perhaps mention that an example of when preserving order is not required is for Properties

Reply:
I'll add an extra paragraph with that.